src/python/pants/backend/codegen/protobuf/python/rules.py

```diff
@@ -127,7 +127,7 @@ async def generate_python_from_protobuf(
                 dist_info.project_name == "mypy-protobuf" and dist_info.version.major >= 2
                 for dist_info in mypy_info
             ):
-                # TODO: merge PEXes together using `pex_path instead once possible through
+                # TODO: merge PEXes together using `pex_path` instead once possible through
```

Comment:
It's possible already. See an example here: 
https://github.com/pantsbuild/pants/blob/0f27e411038977c8a53475de280e5ba59fbaf139/src/python/pants/backend/python/lint/pylint/rules.py#L177-L184

Reply:
Indeed, but a VenvPex does not currently store a Pex field, which is required for pex path to work. It's not super complex for us to add, but outside the scope for Jonas to do and I didn't want to block them on another prework PR. I'll fix it after this lands.